### ISSUE TITLE
Vagrant package repository endpoint returns 404

### ISSUE THREAD

**Issue by Xenov-X** [go-gitea/gitea#21500]
### Description

Pushing packages to other package repositories works as expected, however, the Vagrant repository endpoint returns a 404. This makes it not possible to upload vagrant boxes to the repository.

Tried to replicate on try.gitea.io, however, authentication to any package repository would not work. Suspect this may have been blocked to limit storage requirements? 

## Minimum viable reproduction steps
1. Deploy gitea 1.17.3 using "Basic" docker-compose.yml example [here](https://docs.gitea.io/en-us/install-with-docker/)
2. Create an account
3. Create demo file (Used in place of an actual box file - issue persists when using a valid box file)
```
echo 1 > file.box
```
4. Push to generic repository [Works - uploads successfully]
```
curl -k --user admin123:admin123 \
     --upload-file file.box \  
     http://localhost:3000/api/packages/admin123/generic/test_package/1.0.1/file.bin
```
Response
```
201 Created
```
5. Push to Vagrant repository
```
curl -k --user admin123:admin123 \
     --upload-file file.box \  
     http://localhost:3000/api/packages/admin123/vagrant/vtest/10.1/hyperv.box
```
Response
```
404 Page not found
```

## Relevant log excerpts 
Each of the above uploads creates a single log entry:
```

gitea     | 2022/10/18 10:37:24 [634eba24] router: completed PUT /api/packages/admin123/generic/test_package/1.0.1/file.bin for 172.20.0.1:40728, 201 Created in 23.6ms @ generic/generic.go:67(generic.UploadPackage)


gitea     | 2022/10/18 10:37:33 [634eba2d] router: completed PUT /api/packages/admin123/vagrant/vtest/10.0/hyperv.box for 172.20.0.1:36440, 404 Not Found in 18.2ms @ packages/api.go:56(packages.Routes)

```

### Gitea Version

1.17.3

### Can you reproduce the bug on the Gitea demo site?

No

### How are you running Gitea?

Replicated both on downloaded package installation & docker based install

**Comment by KN4CK3R** [comment]
That's a problem of our not-versioned docs. The vagrant registry is only available in Gitea 1.18.

**Comment by Xenov-X** [comment]
Thanks for getting back to me. Understood!
